docs/config.toml

```diff
@@ -0,0 +1,62 @@
+baseURL = "https://trisatest.net/"
+
+languageCode = "en-us"
+DefaultContentLanguage = "en"
+title = "TRISA"
+theme = "hugo-theme-techdoc"
+
+hasCJKLanguage = true
+metaDataFormat = "yaml"
+
+pygmentsCodeFences = true
+pygmentsUseClasses = true
+
+defaultContentLanguage = "en"
+defaultContentLanguageInSubdir= false
+enableMissingTranslationPlaceholders = false
+
+[Params]
+
+    # Source Code repository section
+    description = "Source code for TRISA implementation"
+    github_repository = "https://github.com/trisacrypto/trisa"
+    version = ""
+
+    # Documentation repository section
+    # documentation repository (set edit link to documentation repository)
+    github_doc_repository = "https://github.com/trisacrypto/trisa"
```

Comment:
Thank you so much for carefully reviewing the config and updating all of these links!